modules/common/go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/openstack-k8s-operators/infra-operator/apis v0.0.0-20230313171323-3a588eef4569
```

Comment:
Yeah, that's a fair point. I'll see where we land with the general discussion about the `GetMemcachedPodList()` method. Maybe we don't use this at all and instead just add some useful methods to the infra-operator instead?

This import was only necessary if we want to create some lib-common methods to create a memcached instance. But if we prefer to avoid the dependencies, we can move some of this logic elsewhere.